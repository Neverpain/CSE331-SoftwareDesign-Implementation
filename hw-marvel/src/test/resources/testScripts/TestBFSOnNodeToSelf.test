# Tests the findPath method on singular node and prints out
# nothing for the path from one node to itself

CreateGraph g

AddNode g n1

FindPath g n1 n1