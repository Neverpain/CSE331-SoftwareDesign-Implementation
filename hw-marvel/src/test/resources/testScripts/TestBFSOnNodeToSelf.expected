# Tests the findPath method on singular node and prints out
# nothing for the path from one node to itself

created graph g

added node n1 to g

path from n1 to n1: